enzyme/Enzyme/CApi.cpp

```diff
@@ -323,6 +323,31 @@ LLVMBasicBlockRef EnzymeGradientUtilsAllocationBlock(GradientUtils *gutils) {
   return wrap(gutils->inversionAllocs);
 }
 
+LLVMValueRef EnzymeCreateDual(EnzymeLogicRef Logic, LLVMValueRef todiff,
```

Comment:
Perhaps CreateForwardDiff ?  Also you probably don't need to pass a derivative mode here (unless we want to leave an option to expand a new vector forward mode).

Reply:
done